docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -47,6 +47,10 @@
 * __Release Date__:  March 13, 2025
 
 ### New Features and Changes
+* __New Analysis Tool:  Single-Cell RNAseq__:
```

Comment:
@siosonel Add a return above this line and I'll be ready to merge this.  I'll approve for now.

Reply:
Done. Thanks for the review!